plugins/kubernetes/test/models/kubernetes/deploy_executor_test.rb

```diff
@@ -424,7 +424,7 @@ def worker_is_unstable
 
         refute execute!
 
-        out.must_include "LOGS:\nLOG-1\n"
+        out.must_match /LOGS:\s+LOG-1/
```

Comment:
I liked the '\n' assertion more ... this might just be a space and would look weird ...